src/theme/docs/components/Colors.tsx

```diff
@@ -0,0 +1,125 @@
+import { Box, Text } from "~/components";
+import { vars } from "~/theme/contract.css";
+import { useVarValues } from "~/utils";
+
+const getColorsBy = (colors: object, field: string) =>
+  Object.entries(colors)
+    .filter(([key]) => key.startsWith(field))
+    .sort();
+
+const ColorItem = ({ sectionName, colors, children }: any) => {
```

Comment:
```suggestion
const ColorItem = ({
  sectionName,
  colors,
  children,
}: {
  sectionName: string;
  colors: [string, string][];
  children: ReactNode;
}) => {
```

Reply:
yeah, I forgot about the typings. thanks!